packages/core/src/lib/highlight-code/prism.ts

```diff
@@ -0,0 +1,25 @@
+/**
+ * Wrapper module for Prism.js
+ */
+
+import PrismPackage from 'prismjs/package.json';
+import Prism from 'prismjs';
+import './viam-prism-theme.css';
+
+/**
+ * We use the prism autoloader to handle loading in language grammar files. The
+ * default path for the grammar files is a CDN link so we don't have to include
+ * grammar files in our bundle. If you prefer to point to your own grammars and
+ * bypass the CDN, define the path to those files.
+ *
+ * See: https://prismjs.com/plugins/autoloader/
+ */
+import 'prismjs/plugins/autoloader/prism-autoloader';
+const grammarsPath = `https://cdnjs.cloudflare.com/ajax/libs/prism/${PrismPackage.version}/components/`;
+
+export const getPrismModule = (): typeof import('prismjs') => {
```

Comment:
Since we're importing it already, I think this can be `typeof Prism`


```suggestion
export const getPrismModule = (): typeof Prism => {
```

Reply:
updated!